.github/workflows/review.yml

```
@@ -0,0 +1,61 @@
+on:
```

`name` could be added:

```suggestion
name: CI
on:
```

And the filename is still `review.yml` and should be named something like `main.yml` (`master` branch has to be renamed in order for it to make sense)